server/routes/v1/validators.js

```diff
@@ -70,8 +72,10 @@ module.exports = (req, res) => {
   }))
 
   if (req.query.unl === process.env.REACT_APP_VALIDATOR) {
-    return res.send(validatorSummary.filter((val) => val.unl === req.query.unl))
+    return res
+      .status(200)
+      .json(validatorSummary.filter((val) => val.unl === req.query.unl))
   }
 
-  return res.send(validatorSummary)
+  return res.status(200).json(validatorSummary)
```

Comment:
Does this ensure that the correct headers are set?

Reply:
I don't think headers are used at all.